spec/features/brexit_checker/accounts_spec.rb

```diff
@@ -3,262 +3,242 @@
 require "gds_api/test_helpers/email_alert_api"
 
 RSpec.feature "Brexit Checker accounts", type: :feature do
+  include GovukAccountHelper
   include GdsApi::TestHelpers::AccountApi
   include GdsApi::TestHelpers::EmailAlertApi
 
-  before do
-    stub_request(:get, Plek.find("account-manager")).to_return(status: 200)
-  end
+  let(:mock_results) { %w[nationality-eu] }
 
-  context "with accounts enabled" do
-    before do
-      allow(Rails.configuration).to receive(:feature_flag_govuk_accounts).and_return(true)
+  context "the user is not logged in" do
+    context "/transition-check/results" do
+      it "shows the normal call-to-action" do
+        given_i_am_on_the_results_page
+        expect(page).to have_content(I18n.t("brexit_checker.results.email_sign_up_title"))
+      end
     end
 
-    let(:mock_results) { %w[nationality-eu] }
-
-    context "the user is not logged in" do
-      context "/transition-check/results" do
-        it "shows the normal call-to-action" do
-          given_i_am_on_the_results_page
-          expect(page).to have_content(I18n.t("brexit_checker.results.email_sign_up_title"))
-        end
+    context "/transition-check/saved-results" do
+      it "redirects to login page" do
+        stub_sign_in_url(redirect_path: "/transition-check/saved-results")
```

Comment:
Is this `stub_sign_in_url` needed?  It should be the user's browser which gets sent to the sign-in URL, finder-frontend shouldn't make a request to it.

Since this is the only test which uses `stub_sign_in_url`, if it's not needed the stub can be deleted, and it's probably better to just put `transition_checker_new_session_url` in this file rather than have a one-method helper.

Reply:
humm perhaps not... tests pass without it!